### PR TITLE
Spelling - Ring of Fire Protection (EN)

### PR DESCRIPTION
**Pull request by AmProsius**
**Describe the bug**
In the English localization the description of the ring *Protection of Fire* is inconsistent and grammatically incorrect.

**Changelog**
Changed description of ring "Protection of Fire" to "Ring of Fire Protection"

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * Fix [#50](https://g1cp.org/issues/50): The inaccessible chest of the crypt below the stonehenge is now correctly positioned and accessible.
 * Fix [#52](https://g1cp.org/issues/52): The grindstone in the New Camp now correctly requires a sword blade to use.
 * Fix [#149](https://g1cp.org/issues/149): The armor "Improved ore Armor" is now correctly labelled as "Improved Ore Armor".
+* Fix [#152](https://g1cp.org/issues/152): The description of the ring "Protection of Fire" is corrected to "Ring of Fire Protection".
```
Comment (szapp) [inline CHANGELOG.md:9]: I updated the text to differentiate that it's the description not the label of the item (compare changelog entries for #149 and #49).

File: src/Ninja/G1CP/Content/Fixes/Session/fix152_EN_ProtectionOfFireDescription.d [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/*
+ * #152 Spelling - Ring of Fire Protection (EN)
+ */
+func int G1CP_152_EN_ProtectionOfFireDescription() {
+    var int symbId; symbId = MEM_GetSymbolIndex("Schutzring_Feuer2");
+    const string needle = "Protection of Fire";
+    const string replace = "Ring of Fire Protection";
+    return (G1CP_ReplaceAssignStr(symbId, "C_ITEM.DESCRIPTION", 0, needle, replace) > 0);
```
Comment (szapp) [inline src/Ninja/G1CP/Content/Fixes/Session/fix152_EN_ProtectionOfFireDescription.d:8]: I have split this line to stay within 120 character limit of the line.

File: src/Ninja/G1CP/Content/Tests/test152.d [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        G1CP_TestsuiteErrorDetail("Item 'Schutzring_Feuer2' not found");
+        return FALSE;
+    };
+
+    // Create the ring locally
+    if (Itm_GetPtr(symbId)) {
+        if (Hlp_StrCmp(item.description, "Ring of Fire Protection")) {
+            return TRUE;
+        } else {
+            var string msg; msg = "Description incorrect: description = '";
```
Comment (szapp) [inline src/Ninja/G1CP/Content/Tests/test152.d:27]: I set "description" to upper case for consistent test output.

**Review by szapp — APPROVED** [review]
I updated a few things. Test passes.

**Comment by m0ezes** [comment]
Same problem with this item exists in the German script: `description = "Schutz vor Feuer";`

Suggested text fix:

A: `"Feuerschutzring"` - to match the description of the other fire prot ring "Flammenschutzring"

or

B: `"Ring des Feuerschutzes"` - to match the naming pattern of most other rings in the game

**Comment by szapp** [comment]
Thanks! (B) might be important here. Consistency between languages is secondary to consistency within a language. A comparison between the descriptions of ring items in the German localization will be essential.

**Pull request merged**